### PR TITLE
simplified test for event transfer success

### PR DESCRIPTION
**Pull request by hackaugusto**
I tried to simplify and remove the `gevent.sleep` from the test, however I'm not 100% sure if the result is better

### CODE REVIEW

File: raiden/tests/integration/test_echo_node.py [hunk 1]
```diff
@@ -17,40 +20,49 @@
...
 ):
-    app0, app1, app2, receiver_app = raiden_chain
+    sender_apps = raiden_chain[:-1]
+    target_app = raiden_chain[-1]
+
     token_address = token_addresses[0]
+    registry_address = target_app.raiden.default_registry.address
+    target_address = target_app.raiden.address
 
-    expected = dict()
```
Comment (hackaugusto) [inline raiden/tests/integration/test_echo_node.py]: This was not being used.

File: raiden/tests/integration/test_echo_node.py [hunk 2]
```diff
@@ -17,40 +20,49 @@
...
+    message_handler = WaitForMessage()
+    target_app.raiden.message_handler = message_handler
 
-    for num, app in enumerate([app0, app1, app2]):
-        amount = 1 + num
-        payment_status = RaidenAPI(app.raiden).transfer_async(
```
Comment (hackaugusto) [inline raiden/tests/integration/test_echo_node.py]: This was being done in serial

**Comment by hackaugusto** [comment]
Huumm, maybe it's just better to remove this test in favor of:

https://github.com/raiden-network/raiden/blob/202e5cae4b3b083b343235c218fdf63771d894f9/raiden/tests/integration/transfer/test_mediatedtransfer_events.py#L16-L20

Which seems easier to read and more complete.

File: raiden/tests/integration/test_echo_node.py [hunk 3]
```diff
@@ -17,40 +20,49 @@
...
+        wait.wait()
+        assert search_for_item(
+            target_app.raiden.wal.storage.get_events(),
+            EventPaymentReceivedSuccess,
+            {
+                'amount': amount,
+                'identifier': amount,
+                'initiator': sender,
+                'payment_network_identifier': registry_address,
+                # 'token_network_identifier': ,
```
Comment (rakanalh) [inline raiden/tests/integration/test_echo_node.py]: Nitpick: this can be removed.

Comment (hackaugusto) [reply raiden/tests/integration/test_echo_node.py]: I left it there to have a full list of all the events attributes, but I can remove it.

What about the other test that is pretty much the same as this?

Comment (rakanalh) [reply raiden/tests/integration/test_echo_node.py]: I think it makes sense for us to remove duplicate tests. 

It's your call.

**Pull request merged**